young_yacks/game/mixins.py

```diff
@@ -0,0 +1,19 @@
+from weakref import WeakSet
+from typing import ClassVar
+
+
+class Queryable:
+    """Adds a Class.all() method to get all in-memory instances of the class."""
+    _instances: ClassVar = WeakSet()
```

Comment:
I've played around in IDLE with this and it doesn't seem to work.

Reply:
Let's see about it.  If needed, I'll write a bunch of tests tonight to show how they work instead of writing other project code.  But you'll need to get more specific here, and besides, this class is meant to be only subclassed.